app/js/controllers/playerListCtrl.js

```diff
@@ -351,7 +347,8 @@ angular.module('KMCModule').controller('PlayerListCtrl',
 										$scope.data.splice($scope.data.indexOf(player), 1);
 										$scope.triggerLayoutChange();
 									}, function (reason) {
-										utilsSvc.alert('Upgrade failure', reason);
+										var msg = reason.replace('\n', '<br/>');
```

Comment:
```suggestion
										var msg = reason.replaceAll('\n', '<br/>');
```
just in case when reason contains more than a single `\n`